### PR TITLE
Projects, about me and contact me sections

### PR DESCRIPTION
**Pull request by PedroH-Neves**
- Created  mobile version of projects, about me and contact sections
- Add more CSS styles
- Change linters errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,7 +57,193 @@ <h3 id="h3" class="primary-color secundary-font">
...
+        <ul class="project-list-tags">
+          <li class="project-list-item"><span class="tag">rust</span></li>
+          <li class="project-list-item"><span class="tag">elixir</span></li>
+          <li class="project-list-item"><span class="tag">c++</span></li>
+          <li class="project-list-item"><span class="tag">lua</span></li>
+        </ul>
+        <a href="#" class="project-link no-underscore">See Project</a>
+      </div>
+      <section class="projects-section-two">
+        <div class="project-box-two">
```
Comment (HAMMAS-SALEEM) [inline index.html:83]: - [x] Please use `display: grid;` to align the cards to follow the requirements :+1:

File: index.html [hunk 2]
```diff
@@ -56,7 +57,193 @@ <h3 id="h3" class="primary-color secundary-font">
...
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+            standard
+          </p>
+          <ul class="project-list-tags-two">
+            <li class="project-list-item-two"><span class="tag">rust</span></li>
+            <li class="project-list-item-two"><span class="tag">elixir</span></li>
+            <li class="project-list-item-two"><span class="tag">lua</span></li>
+          </ul>
+          <a href="#" class="project-link-two no-underscore">See Project</a>
+        </div>
```
Comment (HAMMAS-SALEEM) [inline index.html]: ### Figma Effects
Screenshot from 2022-08-21 05-42-51


- [x] Your see project button should look like similar to Figma template buttons.
- [x] Your buttons should have all the effects as shown in the Figma template. Please add them to follow the Figma template :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @PedroH-Neves  :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Meaningful commits
:heavy_check_mark: Added .gitignore
:heavy_check_mark: Following GitHub Flow

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions
**[Optional]**

- [ ] Your branch name is not descriptive enough. You are using **development**, you can use **feature/portfolio-finish-mobile** :blush: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. Otherwise reach out to me on slack which is more fast. You can reach out to me on slack by searching my name in **full-time-students** channel :+1:

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi <STUDENT_MENTION>, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

goodjob-slowclap


## Good Highlights ✅

- Changes requested by previous reviewer have been implemented
- Meaningful commit messages
- PR is descriptive
- Linter checks are passing

## Optional suggestions

- I recommend that you remove the `demo` section of your README if it is not available (the link provided is not an active link to your demo). When it is available, the `If available` note should be removed too. This will make your documentation more concise and professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**